src/memory/server/MemoryServer.ts

```diff
@@ -48,17 +48,25 @@ export class MemoryServer {
                 </head>
                 <body>
                     <div id="app"></div>
-                    ${this.loadScript(context, 'out/vendor.js')}
-                    ${this.loadScript(context, 'out/MemoryBrowser.js')}
+                    ${this.loadScript(context, 'out/packages.js', newPanel)}
```

Comment:
Non-blocker: You do not need to change 'out/vendor.js' here. Also if you dont change the naming you also do not need to change anything on web.config.js file.

Reply:
Hi @asimgunes, thank for your information